### PR TITLE
Views

### PR DESCRIPTION
**Pull request by vanelnw**
today we :
- Implement the design from the sneak peek wireframes.

### CODE REVIEW

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @vanelnw, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## ❌ Invalid Code Review Request

You have submitted a project that has a failed linter check. Kindly fix your linter errors and submit again 👍🏽 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app/views/shared/_comment.html.erb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<% if (post.comments.empty?) %>
+  <p>No comment yet.</p>
+<% else %>
+  <% post.recent_comments.each do |comment| %>
+    <ul class="border-black ">
+      <li><%= comment.author.name %>: <%= comment.text %></li>
+    </ul>
+  <% end %>
+<% end %>
```
Comment (omarsalem7) [inline app/views/shared/_comment.html.erb]: - [ ] When I click on see all posts button, I got the following error, kindly fix it and make sure your app work properly
>image

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @vanelnw 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Use templates for shared content.✔️
- Use preprocessed HTML files with embedded Ruby code. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/shared/_comment.html.erb (hunk 1 above)
Comment (vanelnw) [reply app/views/shared/_comment.html.erb]: HI @omarsalem7 , thank you for notice that, i will fix it

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @vanelnw 

Your project is complete! Great job for making the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- wireframe design followed
- recent post and comments ok 
- No linter errors



--[optional] please kindy remove the live demo section from your Readme file since you don't yet have a deployed version of the project. nice work

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**